qml/MenuFingerterm.qml

```diff
@@ -18,7 +18,7 @@
 */
 
 import QtQuick 2.0
-import QtQuick.XmlListModel 2.0
+import QtQml.XmlListModel
```

Comment:
This won't work with Qt5.x as version number is mandatory with it. One simple approach could go a follows

- Rename / copy qml/MenuFingerterm.qml to  qml/MenuFingerterm5.qml and qml/MenuFingerterm6.qml
- Have Qt6 changes in qml/MenuFingerterm6.qml and Qt5 in qml/MenuFingerterm5.qml
- Expose Qt major version to the root context or have it as a singleton
- On qml/Main.qml use Loader to correct menu

Something like this to Main.qml to replace MenuFingerterm
```
Loader {
    id: menu

    readonly property showing: item && item.showing

    anchors.fill: parent
    source: QtMajorVersion === 5 ? Qt.resolvedUrl("MenuFingerterm5.qml") : Qt.resolvedUrl("MenuFingerterm6.qml")
}
```

Point is that neither of these menu component won't get compiled when Main.qml is compiled.

Reply:
I wouldn't maybe even copy the whole MenuFingerterm.qml. It should be enough putting the ~10 line  XmlListModel snippet into qml version specific model files and load such on MenuFingerTerm based on qt version.